srcpkgs/pmbootstrap/template

```diff
@@ -1,15 +1,15 @@
 # Template file for 'pmbootstrap'
 pkgname=pmbootstrap
-version=2.0.0
-revision=2
+version=2.1.0
+revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 depends="git openssl python3"
 short_desc="Package build and device flashing tool for postmarketOS"
-maintainer="Jami Kettunen <jami.kettunen@protonmail.com>"
+maintainer="Eloi Torrents <eloitor@disroot.org>"
 license="GPL-3.0-or-later"
 homepage="https://postmarketos.org"
 changelog="https://git.sr.ht/~postmarketos/pmbootstrap/refs"
```

Comment:
I'm fine with the maintainership change :) btw it has moved back to https://gitlab.com/postmarketOS/pmbootstrap following https://postmarketos.org/blog/2024/01/17/moving-pmbootstrap/ so may want to replace all occurances of the sr.ht URLs with something else

Reply:
Thanks for the info, I wasn't aware. I didn't find a release on gitlab yet. I'll make the change on the next update.